.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           - command: check
             args: --all-targets --all-features
           - command: check
-            args: --target thumbv6m-none-eabi -p fuel-asm -p fuel-storage -p fuel-merkle --no-default-features
+            args: --target thumbv6m-none-eabi -p fuel-crypto --no-default-features
```

Comment:
why only fuel-crypto now instead of all the other crates?

Reply:
Good catch. I didn't intend to remove those. Restored in 0a180b40